custom_functions/indicator_remove_tag.json

```diff
@@ -0,0 +1,40 @@
+{
+    "create_time": "2022-05-13T12:04:36.303130+00:00",
+    "custom_function_id": "a888dd1c72bc756c4cb353cef1e5fbafe6e7e61d",
+    "description": "Remove tag(s) from an existing indicator record.",
+    "draft_mode": false,
+    "inputs": [
+        {
+            "contains_type": [
+                "*"
+            ],
+            "description": "Specifies the indicator which the tag will be removed from. Supports a string indicator value or an indicator id.",
+            "input_type": "item",
+            "name": "indicator",
+            "placeholder": "my_indicator"
+        },
+        {
+            "contains_type": [
+                "*"
+            ],
+            "description": "Comma separated list of tags. Tags should only contain characters Aa-Zz, 0-9, '-', and '_'.",
+            "input_type": "item",
+            "name": "tags",
+            "placeholder": "tag1,tag2,...,tagk"
+        }
+    ],
+    "outputs": [
+        {
+            "contains_type": [],
+            "data_path": "indicator_id",
+            "description": "The indicator id that was changed."
+        },
+        {
+            "contains_type": [],
+            "data_path": "indicator_tags",
+            "description": "The new tags for the indicator"
```

Comment:
I think this should be something like "The remaining tags on the indicator after the modification"